util/Migrator/DbScripts/2021-04-27_00_OrganizationUserDataType.sql

```diff
@@ -0,0 +1,26 @@
+-- Create OrganizationUser Type
+IF NOT EXISTS (
+    SELECT
+        *
+    FROM
+        sys.types
+    WHERE 
+        [Name] = 'OrganizationUserType' AND
+        is_user_defined = 1
+)
+CREATE TYPE [dbo].[OrganizationUserType] AS TABLE(
```

Comment:
missing `BEGIN` and `END` keywords within the `IF` block, not sure this will work as-is.